server/pom.xml

```diff
@@ -1,58 +1,94 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>com.google.sps</groupId>
-    <artifactId>Voxetta</artifactId>
-    <version>1</version>
-    <packaging>war</packaging>
-    <properties>
-        <!-- Configured for 1.8 (for now); compiled on 11 -->
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <failOnMissingWebXml>false</failOnMissingWebXml>
-    </properties>
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.google.sps</groupId>
+	<artifactId>Voxetta</artifactId>
+	<version>1</version>
+	<packaging>war</packaging>
+	<properties>
+		<!-- Configured for 1.8 (for now); compiled on 11 -->
```

Comment:
What impact (if any) will the configuration versus compiling version difference make?

Reply:
honestly, no idea